pkg/docker/client.go

```diff
@@ -438,3 +459,11 @@ func (d *Docker) getKindControlPlaneInfo() (types.ContainerJSON, error) {
 
 	return inspect, nil
 }
+
+func (d *Docker) GetServerVersion() (string, error) {
```

Comment:
You should use `mockgen` to generate the file e.g. `mockgen -package fake  github.com/RasaHQ/rasactl/pkg/docker Interface > pkg/docker/fake/mock_client.go`

Reply:
Learn mock now 🏃🏻